src/rotp/lang/fr/labels.txt

```diff
@@ -801,7 +801,7 @@ MAIN_HELP_2K_2F_HOLD              | («Ctrl» pour les drapeaux de droite, «Shi
 MAIN_HELP_2L                      | Le montant de la réserve stockée dans le trésor de l’empire.
 MAIN_HELP_2M                      | L’état actuel de la recherche pour chacun des six domaines technologiques.
 MAIN_HELP_3A                      | Jeu - revient au menu d’ouverture. Vous permet d’ajuster les paramètres ou d’enregistrer, de charger et de démarrer des jeux.
-MAIN_HELP_3B                      | Systèmes - Cartographie de la galaxie indiquant le status des systèmes selon différents critères
+MAIN_HELP_3B                      | Systèmes - Cartographie de la galaxie indiquant le statut des systèmes selon différents critères
```

Comment:
status n'est pas français, mais statut n'est probablement pas le mot correct. Ici et ailleurs.

Reply:
Status is a perfectly cromulent word! 😉
Mais je vais quand même changer la phrase...
Bien vu!